src/main/java/carparks/parking/ParkingService.java

```diff
@@ -0,0 +1,190 @@
+package carparks.parking;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ParkingService {
```

Comment:
Jak byś określił odpowiedzialność tej usługi? Bo z tego co widzę, zarządza ona sesjami parkowania, ale także np. przygotowuje podsumowania dniowe.

Reply:
Faktycznie, ParkingService obsługuje wszystko co jest związane z logiką parkingów. Stwierdziłem, że przy tak małym systemie to nie ma znaczenia, ale rzeczywiście powinienem był zamiast ParkingService stworzyć np ParkingSessionService i ParkingSummaryService i podzielić odpowiedzialność między te 2 klasy.